### ISSUE TITLE
`--bootstrap-version` flag isn't working

### ISSUE THREAD

**Issue by mattse** [chef/knife-google#67]
I'm using this knife plugin to set up GCE servers using chef.  I require that a specific version (12.3.0) of chef-client is installed on the server nodes, because the latest version (12.4.0) breaks my cookbooks.  So I attempted to add the `--bootstrap-version 12.3.0` flag to the `knife google server create` command, but it doesn't work.  The latest chef-client version (12.4.0) gets installed regardless of what I pass in.  Here's the relevant console output:

```
mtse@XXXXXXXXXXXX chef-gce]$ bundle exec knife google server create mtse-base-vanilla-5 -m n1-standard-1 -I centos-7-vXXXXXX --gce-zone XXXXXX -r "role[base-vanilla]" -E staging-gce --bootstrap-version 12.3.0
Looking for Image 'centos-7-vXXXXXX' in Project 'XXXXXXXX
Looking for Image 'centos-7-vXXXXXX' in Project 'centos-cloud'
Found Image 'centos-7-vXXXXX' in Project 'centos-cloud'
Waiting for the boot disk insert operation to complete
Name: mtse-base-vanilla-5
Zone: XXXXXXX
Type: pd-standard
Size: 10 gb
Image: centos-7-XXXXXXX
.
Waiting for the create server operation to complete
.
Waiting for the servers to be in running state
Instance Name: mtse-base-vanilla-5
Machine Type: n1-standard-1
Image: centos-7-vXXXXXXXX
Zone: XXXXXXXXX
Tags: None
Public IP Address: ["XXXXXXXXXX"]
Private IP Address: ["XXXXXXXXXX"]
Waiting for server
Waiting for sshd
..done
Doing old-style registration with the validation key at /home/mtse/.chef/keys/XXXXXXXXXX...
Delete your validation key in order to use your user credentials instead

Connecting to XXXXXXXXXX
XXXXXXXXXX Installing Chef Client...
XXXXXXXXXX   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current

▽
end
XXXXXXXXXX                                  Dload  Upload   Total   Spent    Left  Speed
100 18736  100 18736    0     0  42715      0 --:--:-- --:--:-- --:--:-- 42776
XXXXXXXXXX Downloading Chef 12 for el...
XXXXXXXXXX downloading https://www.opscode.com/chef/metadata?v=12&prerelease=false&nightlies=false&p=el&pv=7&m=x86_64
XXXXXXXXXX   to file /tmp/install.sh.1336/metadata.txt
XXXXXXXXXX trying curl...
XXXXXXXXXX url	https://opscode-omnibus-packages.s3.amazonaws.com/el/7/x86_64/chef-12.4.0-1.el7.x86_64.rpm
XXXXXXXXXX md5	4827ffe781f3736c9e546d1bb462a540
XXXXXXXXXX sha256	4ef1136c0aef6aac052ffe87b46492cd7b1ea95cd221611457904efa0cbf0bba
XXXXXXXXXX downloaded metadata file looks valid...
XXXXXXXXXX downloading https://opscode-omnibus-packages.s3.amazonaws.com/el/7/x86_64/chef-12.4.0-1.el7.x86_64.rpm
XXXXXXXXXX   to file /tmp/install.sh.1336/chef-12.4.0-1.el7.x86_64.rpm
XXXXXXXXXX trying curl...
XXXXXXXXXX Comparing checksum with sha256sum...
XXXXXXXXXX Installing Chef 12
XXXXXXXXXX installing with rpm...
XXXXXXXXXX warning: /tmp/install.sh.1336/chef-12.4.0-1.el7.x86_64.rpm: Header V4 DSA/SHA1 Signature, key ID 83ef826a: NOKEY
XXXXXXXXXX Preparing...                          ################################# [100%]
XXXXXXXXXX Updating / installing...
XXXXXXXXXX    1:chef-12.4.0-1.el7                ################################# [100%]
```

Notice how the `12.4.0` version of chef-client gets installed even though I specify `--bootstrap-version 12.3.0`

**Comment by siddheshwar-more** [comment]
@mattse 
We are looking into this, sorry for delay was busy with releasing stuff !
Is this issue still there for you?

**Comment by Vasu1105** [comment]
@mattse We are able to reproduce the issue. It's not working for command line option but its working if you set this option in knife configuration file i.e. knife.rb, so use configuration file till we fix this issue for command line. 

Thanks.

**Comment by NimishaS** [comment]
@mattse , fix has been merged in master branch. Please verify.

**Comment by mattse** [comment]
@NimishaS I just tested this against master of this repo and it works!  Thanks so much for fixing that.

Are there any plans to do version releases of knife-google?  I see the last version released was 1.3.1 on 4/28/2014.  I'd like to lock down which version of knife-google I'm using, rather than just going off master.
